spec/linter-docker-spec.js

```diff
@@ -0,0 +1,89 @@
+'use babel';
+
+import { join } from 'path';
+// eslint-disable-next-line no-unused-vars
+import { it, fit, wait, beforeEach, afterEach } from 'jasmine-fix';
+
+const fixturePath = join(__dirname, 'fixtures');
+const goodPath = join(fixturePath, 'good', 'Dockerfile');
+const badPath = join(fixturePath, 'bad', 'Dockerfile');
+const badNoFromPath = join(fixturePath, 'bad-no-from', 'Dockerfile');
+const badRepeatedCMDPath = join(fixturePath, 'bad-repeated-cmd', 'Dockerfile');
+const emptyPath = join(fixturePath, 'empty', 'Dockerfile');
+
+describe('The docker provider for Linter', () => {
+  const lint = require('../lib/init').provideLinter().lint;
+
+  beforeEach(async () => {
+    // Info about this beforeEach() implementation:
+    // https://github.com/AtomLinter/Meta/issues/15
+    const activationPromise = atom.packages.activatePackage('linter-docker');
+
+    // await atom.packages.activatePackage('language-python');
```

Comment:
Since this package isn't using `activationHooks`, this whole block could be replaced with just this:
```js
await atom.packages.activatePackage('linter-docker');
```

If you do want to use the `activationHook` compatible method just in case it gets implemented later then the right way to do it would be to activate the associated language **and** open a file of that language. Since opening a file adds ~1 second to each specs time, it's probably best to just go with the above version for now.